crossbeam-channel/src/select_macro.rs

```diff
@@ -1212,3 +1207,14 @@ macro_rules! select {
         )
     };
 }
+
+/// Same as `std::mem::transmute`, except the function is not marked with `unsafe`.
```

Comment:
Pedantically speaking, undefined behavior may be invoked because of the absence of `unsafe`.  Not that I'm opposing to this change, but how about saying it's a `HACK`?